hardware/qcom/Android.mk

```diff
@@ -15,14 +18,14 @@ MASTER_SIDE_CP_TARGET_LIST := msm8996 msm8998 sdm660 sdm845 sm8150
 
 audio-hal := hardware/qcom/audio
 gps-hal := hardware/qcom/gps/sdm845
-ipa-hal := hardware/qcom/data/ipacfg-mgr/sdm845
+ipa-hal := vendor/qcom/opensource/data/ipacfg-mgr/sdm845
 
-display-hal := hardware/qcom/display/sde
+display-hal := vendor/qcom/opensource/display/sde
```

Comment:
Why not just move it do `opensource/display`?
We only chose the `sde` subfolder back then for... I think it was to differentiate from `msmfb`?

Reply:
good suggestion, done